config-bootstrapper/src/main/resources/configs/config-bootstrapper/entity-service/hypertrace/1_entitytypes.conf

```diff
@@ -77,6 +77,26 @@ commands = [
             		}
             	],
             	"name" : "API"
+            },
+            {
+              "attributeType" : [
+                {
+                  "name" : "API_TYPE",
```

Comment:
```suggestion
                  "name" : "BACKEND_API_TYPE",
```

Reply:
discussed offline. we can keep the attributes same as API entity. We can use different constants and methods to segregate backend API accessors